config.schema.json

```
@@ -47,9 +47,24 @@
             "description": "Indicates the number of seconds between pushes to SwitchBot API."
           },
           "debug": {
-            "title": "Debug",
-            "type": "boolean",
-            "description": "Turn plugin's debug mode on."
+            "title": "Logging Setting",
+            "type": "string",
+            "description": "Enble 'Device' or 'Debug' logging.",
```

You're missing an 'a' here 👀 
```suggestion
            "description": "Enable 'Device' or 'Debug' logging.",
```